package.json

```diff
@@ -275,7 +275,7 @@
     "@metamask/rpc-methods": "^3.0.0",
     "@metamask/safe-event-emitter": "^2.0.0",
     "@metamask/scure-bip39": "^2.0.3",
-    "@metamask/selected-network-controller": "^1.0.0",
+    "@metamask/selected-network-controller": "^2.0.0",
```

Comment:
[nit] Just curious, is this required for this PR?

Reply:
selected-network-controller v1.0.0 had a peerDep of network-controller ^12.1.2.

v2.0.0 has a peerDep of 13